binary_transparency/firmware/internal/crypto/signature_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package crypto
+
+import (
+	"testing"
+
+	"github.com/golang/glog"
+)
+
+func TestSignatureRoundTrip(t *testing.T) {
+	for _, test := range []struct {
+		desc       string
+		signbody   string
+		verifbody  string
+		wantStatus bool
```

Comment:
as above, this probably becomes `wantErr`

Reply:
DONE!